Vagrantfile

```diff
@@ -49,12 +49,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
 
     config.vm.provider "virtualbox" do |v|
-        v.memory = 1024
+        v.memory = 2048
```

Comment:
With `celery beat` and `celery worker` running, we quickly run out of memory on our vagrant deploys. I tried bumping this to just 1.5GB, but that still wasn't enough. I'll probably also have to bump up our staging servers.